corehq/apps/app_manager/management/commands/migrate_advanced_form_preload.py

```diff
@@ -0,0 +1,24 @@
+from corehq.apps.app_manager.management.commands.migrate_graph_configuration_properties import AppMigrationCommandBase
```

Comment:
This is great, pulling out the boilerplate. Would be nice to put `AppMigrationCommandBase` somewhere other than `migrate_graph_configuration_properties`, like `corehq.apps.app_manager.management.helpers`, but I certainly won't block the PR on that